netrw/rewire/algebraic_connectivity.py

```diff
@@ -22,12 +22,14 @@ class AlgebraicConnectivity(BaseRewirer):
     Applied Mathematics and computation 219.10 (2013): 5465-5479.
     """
 
-    def full_rewire(self, G, timesteps=-1, tries=100, copy_graph=True, verbose=False):
+    def full_rewire(
+        self, G, timesteps=-1, copy_graph=True, directed=False, verbose=False
+    ):
         """
         Rewire network to maximize algebraic connectivity. In Sydney et al. paper,
         they find that rewiring 30% of the edges is sufficient.
         """
-        return self.step_rewire(G, p, timesteps, tries, copy_graph, verbose)
+        return self.step_rewire(G, timesteps, copy_graph, directed, verbose)
 
     def step_rewire(
         self, G, timesteps=1, copy_graph=True, directed=False, verbose=False
```

Comment:
This should have `copy_graph` as False by default.

Reply:
done. thanks for the catch